djgeojson/serializers.py

```diff
@@ -42,6 +41,11 @@
 from . import GEOJSON_DEFAULT_SRID
 from .fields import GeoJSONField
 
+if django.VERSION >= (5, 2):
+    get_model = PythonDeserializer._get_model_from_node
+else:
+    from django.core.serializers.python import _get_model as get_model
```

Comment:
Handling the move in this commit: https://github.com/django/django/commit/ee5147cfd7de2add74a285537a8968ec074e70cd#diff-d5a29ce7de1d5ea1a712a47adbf91cd8681945d25663098ba8705592cbc4d6d5L188